src/main/java/laddergame/domain/LadderGame.java

```diff
@@ -0,0 +1,62 @@
+package laddergame.domain;
+
+import laddergame.domain.gameelements.*;
+import laddergame.domain.ladder.Ladder;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class LadderGame {
+    private final Players players;
+    private final Ladder ladder;
+    private final Prizes prizes;
+    private final Map<Player, Prize> playerGameResult;
+
+    public LadderGame(Players players, Ladder ladder, Prizes prizes) {
+
+        this.players = players;
+        this.ladder = ladder;
+        this.prizes = prizes;
+
+        playerGameResult = new LinkedHashMap<>();
+    }
+
+    public void playGame() {
+        ladder.climb(players);
+
+        for (Player player : players.getPlayers()) {
+            Position playerPosition = player.getPlayerPosition();
+            Prize prize = findPrizeByPosition(playerPosition);
+
+            playerGameResult.put(player, prize);
+        }
+    }
+
+    public Prize findPrizeByPlayerName(String playerName) {
+        if(playerGameResult.isEmpty()){
```

Comment:
```suggestion
        if (playerGameResult.isEmpty()) {
```
커밋 전에 린트로 검사하는 습관을 가집시다.

Reply:
컨벤션과 관련된 반복적인 피드백.. 많이 신경쓰겠습니다🥲